src/main/scala/org/bitcoins/core/util/Base58.scala

```diff
@@ -55,6 +60,67 @@ trait Base58 extends BitcoinSLogger {
     }
   }
 
+  def encode(hex : String) : String = {
+    val bytes = BitcoinSUtil.decodeHex(hex)
+    encode(bytes)
+  }
+
+  def encode(byte : Byte) : String = encode(Seq(byte))
+
+  /**
+    * Encodes a Base58 address from a hash
+    * @param hash The result of Sha256(RipeMD-160(public key))
+    * @param addressType string. Either "pubkey" or "script"
+    * @param isTestNet boolean
+    * @return
+    */
+  def encodePubKeyHashToBase58Address(hash: Sha256Hash160Digest,
+                                  addressType : String,
+                                  isTestNet : Boolean) : String = {
+    val versionByte : Byte = {
```

Comment:
See comment above, create a function called 'parseVersionByte' or something like that